yb-voyager/cmd/templates/assessmentReport.template

```diff
@@ -75,6 +75,7 @@
         {{end}}
 
         <h2>Database Objects</h2>
+        <p>{{.SchemaSummary.DBObjectsDesc}}</p>
```

Comment:
minor miss, rename `.SchemaSummary.DBObjectsDesc` -> `.SchemaSummaryDBObjectsDesc`
Will fix it in my ongoing PR
cc @makalaaneesh

Reply:
🙏